src/grb_constrs.jl

```diff
@@ -79,7 +75,7 @@ end
 function add_constrs_t!(model::Model, At::Matrix{Float64}, rel::GCharOrVec, b::Vector)
     n, m = size(At)
     (m == length(b) && n == num_vars(model)) || error("Incompatible argument dimensions.")
-    add_constrs_t!(model, sparse(At), rel, b)
+    add_constrs_t!(model, SparseArrays.sparse(At), rel, b)
```

Comment:
`sparse` is used frequently enough that we may want `using SparseArrays`.

Reply:
I've left for now.